spec/lib/Ejercicio1_spec.rb

```diff
@@ -0,0 +1,20 @@
+require_relative '../../lib/Ejercicio1.rb'
+
+describe Ejercicio1 do
+
+  describe '.Words' do
+    o = Ejercicio1.new
+    it "The longest word" do
+      #o.Words(['Hola', 'ComoEstas'])
+        expect(o.Words('Hola', 'ComoEstas', '1234568910')).to eq ('1234568910')
+    end
+
+    it "What happen when two or more words have the same longest" do
```

Comment:
En los `it` se escribe que debería pasar en el caso descripto en el `context`, el cual deberías utilizar, allí iría lo que pusiste como descripción de este`it`